net.sf.VICE.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=X64 (Vice)
```

Comment:
Is it commonly used name for C64 emulator? (Honest question, for a layman it would be confusing.)

Reply:
Good catch. I changed the name to "VICE (C64)" to match the app and the specific emulator mode.